Modules/TRD/TRDQC.json

```diff
@@ -128,10 +128,10 @@
         },
         "taskParameters": {
           "trackType": "0xf",
-          "peakregionstart": "7.0",
-          "peakregionend": "20.0",
-          "pulseheightpeaklower": "1.0",
-          "pulseheightpeakupper": "5.0"
+          "driftRegionStart": "7.0",
+          "driftRegionEnd": "20.0",
+          "peakRegionStart": "1.0",
+          "peakRegionEnd": "5.0"
```

Comment:
could you please also remove the obsolete `taskParameters` from all other tasks please? Looks like they were copy-pasted everywhere without need

Reply:
thanks. you mean from the TrackletsTask.cxx, DigitsTask.cxx, RawData.cxx?? or in the json only??

Reply:
I think its only in the json, right? I'd like the TRDQC.json to be the single file which we can take as reference to configure the QC for production. But please check if there are spurious parameters also in the other tasks which are not used at all. For the DigitsTask for example maybe the regions are also specified for the PH plot based on digits only, I don't know

Reply:
thanks. yes, thats what i was thinking, because there are some params in tasks not used. I will try to make all of them consistent ( with the json as well).

Reply:
hi. removed some params from other tasks as well